source/extensions/http/cache/file_system_http_cache/cache_entry_file.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <memory>
+
+#include "source/extensions/common/async_files/async_file_handle.h"
+#include "source/extensions/common/async_files/async_file_manager.h"
+#include "source/extensions/filters/http/cache/http_cache.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+namespace FileSystemHttpCache {
+
+class FileSystemHttpCache;
+
+class CacheEntryFile {
```

Comment:
is this class used now? If so, where? (couldn't find refs to it in code in the PR).

Reply:
Cleaned that up, along with some other cruft, sorry.